### DISCUSSION TITLE
Theme features not being applied?

### DISCUSSION THREAD

**Discussion by sei-rlehman** [squidfunk/mkdocs-material#4460] [Q&A] [answered]
I am setting up a documentation site for several applications. I want to use the navigation.tabs feature of the theme as used on your documentation site: 
<img width="1390" alt="Screen Shot 2022-10-05 at 11 51 36 AM" src="https://user-images.githubusercontent.com/72103781/194105380-188da8d6-75bc-4dad-b163-d5d47544bc6b.png">

The site that that was generated when I ran `mkdocs new .` had a collapsible navigation panel on the left side. 

I added the following to my mkdocs.yml per the "Setting up navigation" documentation
```
theme: material
  features:
    - navigation.tabs
```

However, this did not seem to take effect and my site still had the collapsible panel navigation element.  After some troubleshooting I decided to pull down your repo and run the site locally. I made no changes to the repo, and I saw the same behavior with the collapsible left panel: 
<img width="1478" alt="Screen Shot 2022-10-05 at 11 58 34 AM" src="https://user-images.githubusercontent.com/72103781/194106808-03dede5d-ec9e-4ada-8d77-cfd5d0a2a1b8.png">

I am running this on Ubuntu 22.04 and using version 8.5.6 of mkdocs-material.

Any idea why I am seeing different navigation behavior on a local instance of your repo versus the public site you have at https://squidfunk.github.io/mkdocs-material/getting-started/

Thank you.

**Comment by sei-rlehman** [comment, marked as answer]
Ok so it was a resolution problem and I feel like a dummy. Increasing the resolution of my development VM on which I was viewing the site caused the navigation tabs to display like I expected.

Is that the expected behavior and is that documented anywhere?

**Reply by squidfunk** [reply]
[From our documentation](https://squidfunk.github.io/mkdocs-material/setup/setting-up-navigation/#navigation-tabs):

> When tabs are enabled, top-level sections are rendered in a menu layer below the header for viewports above `1220px`, but remain as-is on mobile.[1](https://squidfunk.github.io/mkdocs-material/setup/setting-up-navigation/#fn:1) Add the following lines to mkdocs.yml: [...]

Could we make it clearer?

**Reply by sei-rlehman** [reply]
Thanks for the response. I saw the behavior on a default setup before any customization including tabs, so I was a bit confused when I saw that left-side navigation bar pull out - I guess I thought it was a different navigational element that was enabled and not just a response to low resolutions.  Maybe being explicit about the navigation panel's behavior at low desktop resolutions somewhere would help. Or just leave it as is because who would have such a low-res desktop anyway :/ 

Great product btw - exactly what I needed.
